### PR TITLE
10141 add user account UUID to new in progress forms

### PR DESCRIPTION
**Pull request by bosawt**
## Description of change
This PR makes it so that we add a user_account.uuid in the most common places that we create in progress forms

## Original issue(s)
department-of-veterans-affairs/vets-api#10141

## Things to know about this PR
- Navigate anywhere that would create an in progress form
- Confirm on the backend that this in progress form has a user_account_id value

### CODE REVIEW

File: db/migrate/20220628222858_add_user_account_uuid_to_in_progress_forms.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class AddUserAccountUuidToInProgressForms < ActiveRecord::Migration[6.1]
+
+  def change
+    safety_assured do
+      add_reference :in_progress_forms, :user_account, type: :uuid, foreign_key: true, null: true, index: true
```
Comment (LindseySaari) [inline db/migrate/20220628222858_add_user_account_uuid_to_in_progress_forms.rb:5]: You'll have to break out any /db and schema changes to a separate PR

Comment (bosawt) [reply db/migrate/20220628222858_add_user_account_uuid_to_in_progress_forms.rb]: 👍 Rebased on top of the merged database migration, so db/schema is out of this PR now

Comment (LindseySaari) [reply db/migrate/20220628222858_add_user_account_uuid_to_in_progress_forms.rb]: Thanks! GHA appears to be a bit wonky (having issues today) could you re-run the CI pipeline?

**Review by LindseySaari — APPROVED** [review]
Looks good 👍 Migrations are all set in all envs

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**